week1/citibike.sh

```diff
@@ -4,20 +4,20 @@
 #
 
 # count the number of unique stations
-
+cut -d, -f5 201402-citibike-tripdata.csv | sort | uniq | wc -l
 # count the number of unique bikes
-
+cut -d, -f12 201402-citibike-tripdata.csv | sort | uniq | wc -l
 # count the number of trips per day
-
+cut -d, -f2 201402-citibike-tripdata.csv | cut -c2-11 | sort -n | uniq -c
 # find the day with the most rides
-
+cut -d, -f2 201402-citibike-tripdata.csv | cut -c2-11 | sort -n | uniq -c
 # find the day with the fewest rides
-
+cut -d, -f2 201402-citibike-tripdata.csv | cut -c2-11 | sort -n | uniq -c
 # find the id of the bike with the most rides
-
+cut -d, -f12 201402-citibike-tripdata.csv | sort -n | uniq -c | sort
 # count the number of rides by gender and birth year
-
+cut -d, -f14-15 201402-citibike-tripdata.csv | sort | grep '[0-9]' | uniq -c | sort
 # count the number of trips that start on cross streets that both contain numbers (e.g., "1 Ave & E 15 St", "E 39 St & 2 Ave", ...)
-
+cut -d, -f5,9 201402-citibike-tripdata.csv | grep '[0-9]' | cut -d, -f1 | grep "&" | wc -l
 
 # compute the average trip duration
```

Comment:
I can give you a hint on this one. First separate out just the data you want, then clean it up so that it can be easily understood by a number parser. Use awk for arithmetic (using awk for arithmetic is generally a good strategy in the Unix CLI environment).